x-pack/test/security_solution_endpoint/apps/endpoint/policy_list.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { IndexedHostsAndAlertsResponse } from '@kbn/security-solution-plugin/common/endpoint/index_data';
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { PolicyTestResourceInfo } from '../../services/endpoint_policy';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const browser = getService('browser');
+  const pageObjects = getPageObjects([
+    'common',
+    'endpoint',
+    'policy',
+    'endpointPageUtils',
+    'ingestManagerCreatePackagePolicy',
+    'trustedApps',
+  ]);
+  const testSubjects = getService('testSubjects');
+  const policyTestResources = getService('policyTestResources');
+  const endpointTestResources = getService('endpointTestResources');
+
+  describe('When on the Endpoint Policy List Page', () => {
+    before(async () => {
+      const endpointPackage = await policyTestResources.getEndpointPackage();
+      await endpointTestResources.setMetadataTransformFrequency('1s', endpointPackage.version);
+      await browser.refresh();
+    });
+
+    describe('with no policies', () => {
+      it('shows the empty page', async () => {
+        await pageObjects.policy.navigateToPolicyList();
+        await testSubjects.existOrFail('emptyPolicyTable');
+      });
+      it('navigates to Fleet to add the Endpoint Security integration', async () => {
```

Comment:
If we don't add the integration here maybe this should say something like:
`navigates to Fleet and ensures the integration page is loaded correctly`

What do you think?

Reply:
yes I think that wording is more specific! i can update it